setup.cfg

```diff
@@ -9,3 +37,11 @@ with-xunit=1
 source-dir = docs
 build-dir  = sphinx-build
 all_files  = 1
+
+[extras]
+docs =
+    sphinx
+    sphinx_rtd_theme
+tests =
+    oslotest
+    nose==1.3.7
```

Comment:
Let's remove these `extras`, we should not duplicate requirements in different places